tests/align/test_one_operator.sh

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
```

Comment:
To avoid code duplication, I think we should either call this script from `tests/align/test_all_operators.sh` or remove it

Reply:
thanks, removed

Reply:
yes, I'm thinking that we should remove this file, unless we decide to use it as a wrapper and call it from the main script for each test

Reply:
removed